docs/modules/ROOT/pages/architecture.adoc

```diff
@@ -60,25 +64,39 @@ to transfer environments, features and service accounts around the network to fe
 === Dacha
 
 Dacha is where the data that is required by every SDK is cached, and you need at least one of these for an operational
-FeatureHub system. It can be run in-process (using the _Party Server_ design), or separately. In any case, it always
-communicates via NATS, and when it starts it broadcasts for another complete cache and will fill itself from that. If
-none is available, it negotiates a master, and will request the MR to provide details. Each Dacha is defined with a name,
-and the self hosted one is always called "default". All but the very largest of corporations should never need more
-than one cache name, as it is a form of sharding. Each cache is able to handle thousands of environments and their
-features.
+FeatureHub system. It can be run in-process (using the _Party Server_ design), or separately. Edge always talks to
+Dacha which holds permissions, environments, features, and pre-calculated etags for appropriate requests.
+
+==== Architectural Choices for Dacha
+
+There are two choices for Dacha, the standard for Dacha before 1.5.9 is Dacha1.
```

Comment:
I would say: There are two choice for Dacha: Dacha1 and Dacha2 (available from 1.5.9)